.github/workflows/ci.yml

```
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.9", "3.10"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

```suggestion
        python-version: ["3.9", "3.10", "3.11"]  # see #590 for "3.12"
```
We should comment this out and get the PR merged, and once the 3.12 issues are fixed we can add it back.